instrumentation/khttp-0.1/src/test/groovy/KHttpClientTest.groovy

```diff
@@ -21,7 +21,9 @@ class KHttpClientTest extends HttpClientTest {
 
   @Override
   int doRequest(String method, URI uri, Map<String, String> headers, Closure callback) {
-    def response = KHttp.request(method, uri.toString(), headers, Collections.emptyMap(), null, null, null, null, 1)
+    // khttp applies the same timeout for both connect and read
+    def timeout = Math.max(CONNECT_TIMEOUT_MS, READ_TIMEOUT_MS)
+    def response = KHttp.request(method, uri.toString(), headers, Collections.emptyMap(), null, null, null, null, timeout)
```

Comment:
If I'm not mistaken the timeout for khttp is seconds so now we're setting it to 2000s - yeah that'll keep the flakes out ;) But probably meant to convert to secs.

Also if the value isn't ever used in assertions, perhaps can only have one timeout in the base class since it's not such a big deal for us to distinguish them when executing requests in tests.

Reply:
https://khttp.readthedocs.io/en/latest/user/quickstart.html#timeouts

Reply:
opened #593, it looks to me like we don't need the read timeout at all